docs/nydus-failover-upgrade.md

```diff
@@ -0,0 +1,67 @@
+# Nydus Failover and Hot Upgrade
+
+Currently, nydusd supports failover and hot upgrade under fusedev and fscache modes, specifically:
+
+- Nydusd can recover automatically from crashes. After nydusd crashes then recovers, it can still serve container I/O operations.
```

Comment:
Nydusd can automatically recover from crashes and continue serving container I/O operations.